### ISSUE TITLE
Unklare Formulierung Konfiguration von Overbooking im Location-Post

### ISSUE THREAD

**Issue by datengraben** [wielebenwir/commonsbooking#1219]
![image](https://user-images.githubusercontent.com/105302830/232113551-def89da2-b79f-4d23-97d8-8eec09365d57.png)

Beide Punkte sprechen eigentlich vom gleichen Ding, verwenden aber unterschiedliche Begriffe dafür.

`If selected, all not selected [...]`. Anstatt bei der Booking-Range von selected days zu sprechen, denke ich wäre es klarer auch von einer range/timeframe zu sprechen. 

Vorschlag:
1. `If selected, all locked/non-unbookable in any bookable timeframe that is connected to this location can be overbooked. Read the documentation [Create Locations](https://commonsbooking.org/?p=435) for more information on overbooking.`
2. `If selected, overbooked non-bookable days are deducted/subtracted from the number of maximum selectable days of the bookable timeframe.`

**Comment by hansmorb** [comment]
![image](https://user-images.githubusercontent.com/6433480/232208935-64b61fed-92e1-4db9-ae19-f5a7e4dbd1c3.png)

Ich finde in diesem Fall die deutsche Version mal wieder klarer.

Näher an der deutschen Version wäre:

"If selected, days that are not activated can be overbooked when booking. This applies to all time frames of the type "bookable" that are associated with this location. "

**Comment by genevievecory** [comment]
Habe ich das richtig verstanden?

**Allow booking through lockout days**
This checkbox allows the item to be kept out over days when the station is otherwise unreachable. For example, an item may be booked Saturday-Monday though the station is closed on Sundays.  

**Subtract lockout days from total booking days**
Check this box to make each lockout day included in a booking count towards the total duration of a booking. An item booked Saturday-Monday (when the station is closed on Sundays) will count as a 3-day booking, because Sunday will count as a booked day.

**Comment by hansmorb** [comment]
> **Allow booking through lockout days** This checkbox allows the item to be kept out over days when the station is otherwise unreachable. For example, an item may be booked Saturday-Monday though the station is closed on Sundays.
> 
Ja, das sollte eigentlich logischerweise sowohl Holidays als auch nicht definierte Tage betreffen. Bin mir aber unsicher wie man das besser für Endnutzende formulieren kann.
> **Subtract lockout days from total booking days** Check this box to make each lockout day included in a booking count towards the total duration of a booking. An item booked Saturday-Monday (when the station is closed on Sundays) will count as a 3-day booking, because Sunday will count as a booked day.
> 
Ich bin nicht mit dem Wort "lockout day" so happy weil das ja eher nach unüberbuchbarem Holiday oder einer Restriction oder so klingt.
> I'd prefer to avoid the term "overbooking" because it already means something else. If your flight is overbooked, that means more seats have been sold than are on the plane...

**Comment by datengraben** [comment]
@hansmorb 

> "If selected, days that are not activated can be overbooked when booking. [...]"

Wobei ich mich "activated" störe. Habe ich in dem Kontext auch nicht in der PO-FIle finden können. 
Alternative: `not available/bookable`

@genevievecory 

> I'd prefer to avoid the term "overbooking" because it already means something else. If your flight is overbooked, that means more seats have been sold than are on the plane...

Alternative: `Extend`, `overlay` oder `overlap`

> I'd prefer to avoid the term "overbooking" because it already means something else. If your flight is overbooked, that means more seats have been sold than are on the plane...

I like that too, you also can achieve that when you try to talk more technically about the booking duration. See my answer above.

@hansmorb 

> Ja, das sollte eigentlich logischerweise sowohl Holidays als auch nicht definierte Tage betreffen. Bin mir aber unsicher wie man das besser für Endnutzende formulieren kann.

Was heißt nicht definierte Tage?

> Ich bin nicht mit dem Wort "lockout day" so happy weil das ja eher nach unüberbuchbarem Holiday oder einer Restriction oder so klingt.

Ich wäre auch dafür entweder bei lockday zu bleiben, oder auf closing day zu wechseln.

### Vorschlag

Lockout days zu locking days und einen Satz zu @genevievecory s Vorschlag hinzuzufügen.

**Allow booking through locking days** This checkbox allows the item to be kept out over days when the station is otherwise unreachable. For example, an item may be booked Saturday-Monday though the station is closed on Sundays. This applies to any locking day and therefore also includes holidays.

Beid diesem habe ich den ersten Satz mit `include` umgestellt und auch locking days benutzt.

**Subtract locking days from total booking days** Check this box to include each locking day in a booking count towards the total duration of a booking. An item booked Saturday-Monday (when the station is closed on Sundays) will count as a 3-day booking, because Sunday will count as a booked day.

**Comment by hansmorb** [comment]
Bitte nochmal prüfen ob das mit dem beheben von #702 immer noch so uneindeutig ist.

**Comment by hansmorb** [comment]
Keine Antwort, keine Beschwerde.
